src/test/java/org/jenkinsci/plugins/workflow/DSLTest.java

```diff
@@ -148,6 +148,30 @@ public void nonexistentFunctions() throws Exception {
         r.assertLogContains("constructible with compass and straightedge", b);
     }
 
+    /**
+     * Tests the ability to execute a step with an unnamed monomorphic describable argument.
+     */
+    @Issue("JENKINS-29711")
+    @Test
+    public void monomorphic() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "mon");
+        p.setDefinition(new CpsFlowDefinition("monomorphStep firstArg:'one', secondArg:'two'"));
```

Comment:
Also, turns out to be a lot harder to get it to exclude parentheses and brackets for cases like `metamorphStep` but not ones featuring `$class` as a key. So...I say let's just leave it as is, with `metamorphStep([firstArg:'one', secondArg:'two'])` as the output. It ain't the most aesthetically pleasing, but it *works*, so...